elvis.config

```diff
@@ -16,7 +16,6 @@
                   {elvis_style, nesting_level, #{level => 3}},
                   {elvis_style, god_modules, #{limit => 35}},
                   {elvis_style, no_if_expression},
-                  {elvis_style, invalid_dynamic_call},
```

Comment:
Almost all modules (at least those with actual code, i.e. not just behaviour definitions) in this repo will use dynamic calls… it's kinda the point here :)